docs/using.rst

```diff
@@ -114,22 +70,23 @@ To use ADB with Host-Based tools:
     config.servicemanager.services = 'openshift'
 
    **Note:**
-* Docker is a default service for the ADB and does not require any configuration to ensure it is started.
-  Additionally, the Red Hat Enterprise Linux Container Development Kit, which is
-  based on the Atomic Developer Bundle, automatically starts OpenShift as well.
-* You can enable multiple services as a comma separated list. For instance: `docker, openshift`.
 
-3. Enable any specific options for the services you have selected as:
+   * Docker is the default service for ADB and does not require any configuration to ensure it is started.
+     Red Hat Enterprise Linux Container Development Kit, which is based on
+     Atomic Developer Bundle, automatically starts OpenShift.
+   * You can use a comma-separated list to enable multiple services. For instance: `docker, openshift`.
+
+3. Enable the relevant option for the services you have selected in the Vagrantfile as:
 
-   For instance, in OpenShift, specific versions can be specified using the following variables:
+   For example, in OpenShift, specific versions can be specified using the following variables:
```

Comment:
"For example, specific versions of OpenShift can be specified using the following variables:"
?